content/news/2015/02/2015-02-02-agency-perspectives-on-personas-use-development-and-challenges.md

```diff
@@ -2,65 +2,74 @@
 slug: agency-perspectives-on-personas-use-development-and-challenges
 date: 2015-02-02 10:00:00 -0400
 title: Agency Perspectives on Personas (Use, Development and Challenges)
-summary: 'Personas are tools your agency can use to learn about your end users and drive decisions. Personas are so useful because they serve as a communication tool for your team. You can keep these personas in mind to guide any work that your agency performs. Let&#8217;s delve a bit deeper into personas and review two'
-authors:
-  - kdaniel
+summary: Personas are tools your agency can use to learn about your end users
+  and drive decisions. Personas are so useful because they serve as a
+  communication tool for your team. You can keep these personas in mind to guide
+  any work that your agency performs. Let&#8217;s delve a bit deeper into
+  personas and review two
+# See all topics at https://digital.gov/topics
 topics:
   - HHS
   - personas
   - US Department of Agriculture
   - united-states-department-of-health-and-human-services
   - USDA
   - user-experience
+# See all authors at https://digital.gov/authors
+authors:
+  - kdaniel
+# Controls how this page appears across the site
+# 0 -- hidden
+# 1 -- visible
+weight: 1
 ---
-
 Personas are tools your agency can use to learn about your end users and drive decisions. Personas are so useful because they serve as a communication tool for your team. You can keep these personas in mind to guide any work that your agency performs.
 
-Let&#8217;s delve a bit deeper into [personas]({{< ref "2015-01-09-personas-101.md" >}}) and review two examples from the federal community. Below, we have personas from the Department of Human and Health Services (HHS) and the United States Department of Agriculture (USDA).
+Let&#8217;s delve a bit deeper into \[personas]({{< ref "2015-01-09-personas-101.md" >}}) and review two examples from the federal community. The Department of Human and Health Services (HHS) and the United States Department of Agriculture (USDA) both 
```

Comment:
Removed HTML code for apostrophe and slash before the link (it was causing it to render incorrectly).

```suggestion
Let's delve a bit deeper into [personas]({{< ref "2015-01-09-personas-101.md" >}}) and review two examples from the federal community. The Department of Human and Health Services (HHS) and the United States Department of Agriculture (USDA) both 
```

---

Is it supposed to end on `both`?
![image](https://user-images.githubusercontent.com/3385219/210648031-28c9d979-b95d-454e-ac9d-cc6148749acd.png)

Reply:
You're right! It isn't, good catch.